patroni/api.py

```diff
@@ -140,6 +144,25 @@ def do_GET(self, write_status_code_only=False):
             elif path in ('/async', '/asynchronous') and not is_synchronous:
                 status_code = replica_status_code
 
+        # check for user defined tags in query params
+        if not request_for_master and status_code == 200:
```

Comment:
Well, the standby-leader should be also excluded, therefore, these [changes](https://github.com/zalando/patroni/pull/1964/files#diff-13329cd3342d1757c2c9b4dd7a26c0170f4deee5d32eff5c2fd719cc9ff9375bR125-R132) more or less unnecessary.